cmd/lokomotive-controller/main.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/golang/glog"
+
+	controller "github.com/kinvolk/lokomotive/internal"
+)
+
+const (
+	port = "8080"
+)
+
+var tlscert, tlskey string
+
+func usage() {
+	flag.PrintDefaults()
+	os.Exit(0)
+}
+
+func returnError(msg string, err error) {
+	glog.Fatalf("%s: %v", msg, err)
+}
+
+func main() {
+	flag.Usage = usage
+
+	err := flag.Set("logtostderr", "true")
```

Comment:
It seems odd to me to pass the flag values here. Shouldn't we be able to specify those flags via CLI options?